packages/ghostscript.rb

```diff
@@ -1,42 +1,34 @@
 require 'package'
 
 class Ghostscript < Package
-  description 'Ghostscript is the name of a set of software that provides an interpreter for the PostScript language and the PDF file format.'
-  homepage 'https://www.gnu.org/software/ghostscript/'
-  version '9.14.1-1'
+  description 'Interpreter for the PostScript language'
+  homepage 'https://www.ghostscript.com/'
+  version '9.52'
```

Comment:
I tried upgrading this awhile back without success on all the architectures.  I can certainly try again but this may need to be reverted if I'm not successful.

Reply:
Understood, let me know how the compiling goes.